schema/changelog-4.17.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd"
+  logicalFilePath="changelog-4.17">
+
+  <changeSet author="author" id="changelog-4.17">
+
+    <renameColumn tableName="tc_orders" oldColumnName="toAddress" newColumnName="toaddress" />
+    <renameColumn tableName="tc_orders" oldColumnName="fromAddress" newColumnName="fromaddress" />
```

Comment:
I think it might be better to remove and add columns back because depending on the database it might not recognize this as a change.

Reply:
Wouldn't remove columns also remove data?

Reply:
Yes, but it's not used yet.

Reply:
Another option is to rename to some temporary names. Yes, that's probably even better.

Reply:
ah, you can comment faster than I can squash commits.

Reply:
done